server/incore-common/src/main/java/edu/illinois/ncsa/incore/common/utils/JsonUtils.java

```diff
@@ -49,4 +61,149 @@ public static String extractValueFromJsonString(String inId, String inJson) {
             return outList;
         }
     }
+
+    public static JSONObject createUserFinalQuotaJson(String username, IUserFinalQuotaRepository finalQuotaRepository) throws ParseException{
+        UserFinalQuota quota = finalQuotaRepository.getQuotaByUsername(username);   // get default allocation
```

Comment:
This quota is always empty when I test. I checked that the mongo "View" has the right information, but endpoint returns wrong info. I see that they are always coming from the AllocationConstants
<img width="1444" alt="image" src="https://user-images.githubusercontent.com/59935416/159261201-3ba17203-6596-4bce-bad1-25cdd10b42c8.png">
<img width="721" alt="image" src="https://user-images.githubusercontent.com/59935416/159261394-9cd605a0-32c1-4519-9724-117033c42968.png">

Reply:
This has been fixed. Thank you for catching it.